### ISSUE TITLE
Missing "total" if ConsentInterceptor in place

### ISSUE THREAD

**Issue by der-brecher** [jamesagnew/hapi-fhir#1507]
I'm not sure if this is actually a bug since it is not documented. 

Having a ConsentInterceptor configured, I want to i) hide some resources that are unaccessible to the current user (this is checked via meta tags), and ii) hide the meta tags that I use to implement my access control. 

I have two questions concerning this:

1. The interceptor works fine but the total number of records for a very search is not shown. I was wondering if this is intentional.
2. Is a ConsentInterceptor the right place to implement access control of resources depending on meta tags, i.e., to implement resource excludes or is there a better way to do?

**Comment by jamesagnew** [comment]
FYI: Please raise questions on the HAPI FHIR [Google Group](https://groups.google.com/forum/#!forum/hapi-fhir). You can find details on where to get help here: https://github.com/jamesagnew/hapi-fhir/wiki/Getting-Help

This is deliberate, as the total number of resources could be used to "leak" information about resources that had been hidden from the results. Is that not a concern here?

**Comment by der-brecher** [comment]
Absolutely. Concerning the unfiltered resources this is perfectly fine. 
I thought about "total" as the number of entries within the searchset, i.e., total = number of all resources for the provided request - number of hidden entries.
